### ISSUE TITLE
Found 1 proofs - GUI show ZERO coins

### ISSUE THREAD

**Issue by gamilevin** [Spare-Network/spare-blockchain#103]
Hi,

I have 1014 plots, farming for over a week and no winnings.
As my estimated time to win was 1-2 days I started digging the logs.

I saw several proofs found in the file but the UI show zero.

I'd like to understand why am I not getting the coins

Here’s the log (removed sensitive info):

2021-07-06T13:52:05.412 harvester spare.harvester.harvester: INFO     3 plots were eligible for farming XXXXXXXXXXX Found 1 proofs. Time: 3.39201 s. Total 1014 plots
2021-07-06T13:52:05.450 full_node spare.full_node.mempool_manager: INFO     Starting to make block, max cost: 11000000000
2021-07-06T13:52:05.451 full_node spare.full_node.full_node: WARNING  Farmer Puzzle Hash: XXXXXXXXXXX
2021-07-06T13:52:05.451 full_node spare.full_node.full_node: WARNING  Farmer: Pool Target: {'max_height': 0, 'puzzle_hash': 'XXXXXXXXXXX'}
2021-07-06T13:52:05.452 full_node spare.full_node.full_node: WARNING  Farmer: Request: {'challenge_chain_sp': 'XXXXXXXXXXX',
 'challenge_chain_sp_signature': XXXXXXXXXXX',
 'challenge_hash': 'XXXXXXXXXXX',
 'farmer_puzzle_hash': 'XXXXXXXXXXX',
 'pool_signature': 'XXXXXXXXXXX',
 'pool_target': {'max_height': 0, 'puzzle_hash': 'XXXXXXXXXXX'},
 'proof_of_space': {'challenge': 'XXXXXXXXXXX',
                    'plot_public_key': 'XXXXXXXXXXX',
                    'pool_contract_puzzle_hash': None,
                    'pool_public_key': 'XXXXXXXXXXX',
                    'proof': 'XXXXXXXXXXX',
                    'size': 32},
 'reward_chain_sp': 'XXXXXXXXXXX',
 'reward_chain_sp_signature': 'XXXXXXXXXXX',
 'signage_point_index': 3}
2021-07-06T13:52:05.453 full_node spare.full_node.full_node: INFO     Starting to make the unfinished block
2021-07-06T13:52:05.457 full_node spare.full_node.full_node: INFO     Made the unfinished block
2021-07-06T13:52:05.503 full_node spare.full_node.full_node: INFO      Farmed unfinished_block XXXXXXXXXXX, SP: 3, validation time: 0.013962507247924805, cost: 0
2021-07-06T13:52:06.987 full_node full_node_server        : INFO     Connected with full_node {'host': '27.127.66.79', 'port': 7444}
2021-07-06T13:52:06.989 full_node full_node_server        : INFO     Connection closed: 27.127.66.79, node id: XXXXXXXXXXX
2021-07-06T13:52:06.990 full_node spare.full_node.full_node: INFO     peer disconnected {'host': '27.127.66.79', 'port': 7444}
2021-07-06T13:52:07.598 full_node full_node_server        : INFO     Cannot connect to host 187.58.191.222:9444 ssl:<ssl.SSLContext object at 0x000002016605F0B8> [Connect call failed ('187.58.191.222', 9444)]
2021-07-06T13:52:08.928 full_node spare.full_node.full_node: INFO       Finished signage point 4/64: CC: XXXXXXXXXXX RC: XXXXXXXXXXX

**Comment by PC101guy** [comment]
is your wallet fully synced?  you can tell this by looking at the height listed in your wallet and in the full node, they should both be the same for example both should be 85,790.  if your wallet isn't synced up correctly you will not see the true balance.

**Comment by gamilevin** [comment]
Yes, the wallet is synced. I had 3 times "1 proof found"  in the log file in the last 2 days but still no coins.

**Comment by Namsoyo** [comment]
maybe youre hacked, try to make a new receive address (Wallet - Receiv Address - New Address)
add new address unter your Farm the 3 dots on the right top side - Manage Farming Rewards end enter it twice

**Comment by OldButtKicking** [comment]
Are these NFT plots? I stopped using spare farmer with NFT plots and only run with OG plots because of this error. Once I stopped using NFT the error stopped.
